CriticalMass/GetLocationsAndChatMessagesRequest.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  GetLocationsAndChatMessagesRequest.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 11.08.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import Foundation
+
+struct GetLocationsAndChatMessagesRequest: APIRequestDefining {
+    typealias ResponseDataType = ApiResponse
+    var baseUrl: URL
+    var paths: [String]
```

Comment:
```suggestion
    var paths: [String] = []
```

Reply:
I added a struct `Endpoint` that solves the issue I think. I also updated the creation logic of the request